src/hotspot/cpu/aarch64/stubGenerator_aarch64.cpp

```diff
@@ -4669,6 +4667,7 @@ class StubGenerator: public StubCodeGenerator {
     const Register tmp5  = r14;
     const Register tmp6  = r15;
     const Register tmp7  = r16;
+    const Register tmp8  = r17;
```

Comment:
It looks like tmp8 is never used.  The call to multiply_to_len() below needs to be adjusted.